docs/environments/icp.md

```diff
@@ -0,0 +1,49 @@
+# IBM Cloud Private Deployment Guide
+
+Federation v2 can be deployed to and manage [IBM Cloud Private](https://www.ibm.com/cloud/private) clusters.
+As Federation v2 requests Kubernetes v1.11 or greater, so please make sure deploy IBM Cloud Private 3.1.1
```

Comment:
`s/requests/requires/`
`s/so please make sure deploy/please make sure to deploy/`